src/main/java/com/novoda/notils/exception/DeveloperError.java

```diff
@@ -12,7 +12,12 @@ public DeveloperError(String detailMessage) {
         super(detailMessage);
     }
 
+    public DeveloperError(String messageTemplate, String... args) {
```

Comment:
So this means we should avoid calling `.toString()` when assigning the arguments because the formatter might pick another object representation, like formatTo.